### PR TITLE
POST /add-item Endpoint related #38

### PR DESCRIPTION
**Pull request by Hadeel-Salamin**
- [x] install airtable module

- [x] set up Airtable with your own API Key

- [x] get userAirtableId from req and the other details from req.body

- [x] add new item record to Airtable

- [x] if recorded added response with success true otherwise response with success false.

- [x] add /add-item route to the router and pass auth middleware to it.

### CODE REVIEW

**Review by FidaaHersh — APPROVED** [review]

**Review by amanshawar — APPROVED** [review]

File: server/controllers/addItem.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const Airtable = require('airtable');
+
+const { Airtable_API_KEY } = process.env;
+const base = new Airtable({ apiKey: Airtable_API_KEY }).base('appAZnpLnWP0wjAc6');
```
Comment (orjwan98) [inline server/controllers/addItem.js:4]: What is the string inside the brackets in the following line:

```
.base('appAZnpLnWP0wjAc6')

```

Comment (Hadeel-Salamin) [reply server/controllers/addItem.js:4]: base code
Airtable provide you with this, it's like a key for your workspace and does not matter if showen

**Review by jema28 — APPROVED** [review]

File: server/controllers/addItem.js (hunk 1 above)
Comment (orjwan98) [reply server/controllers/addItem.js:4]: cool c:

**Review by orjwan98 — APPROVED** [review]
:+1:

**Pull request merged**